streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -920,6 +921,7 @@ private void closeNonAssignedSuspendedStandbyTasks() {
                 } catch (Exception e) {
                     log.error("{} Failed to remove suspended task standby {}", logPrefix, suspendedTask.getKey(), e);
                 } finally {
+                    task.closeStateManager(false);
```

Comment:
task.close already catches any exceptions and closes the state manager. Do we need to catch it again?